src/HeaderCell.tsx

```diff
@@ -85,11 +85,25 @@ export default function HeaderCell<R, SR>({
       return;
     }
 
+    const colSpan = currentTarget.ariaColSpan;
+
+    function getColSpanColumnsWidth() {
+      let colSpanColumnsWidth = 0;
+      if (colSpan === null) return colSpanColumnsWidth;
+      for (let index = 1; index < Number(colSpan); index++) {
+        // nextElementSibling won't be null as all the colSpan columns are rendered
+        colSpanColumnsWidth += currentTarget.nextElementSibling!.getBoundingClientRect().width;
```

Comment:
This just gets the next column's width
1. why? the next column is unrelated, it just doesn't work well
2. this won't work if there's no column on the right

![Recording 2022-09-21 at 11 13 19](https://user-images.githubusercontent.com/567105/191478900-04b9bb89-eb2a-486f-b245-65ac8ae8f807.gif)

Reply:
Yeah, this is not correct. I wanted to put this PR back to draft